scripts/batch-png-to-svg.sh

```
@@ -2,32 +2,34 @@
 
 # Right now this method only works with DE's simple icon PNGs so I wouldn't try this with
 # anything more complex then an icon. Even then the original would have to be something >= 256x256
-# otherwise you could have misalinged lines and basically blobs
+# otherwise you could have misalinged lines and blobs
+
+indir=$1
+outdir=$2
+
+mkdir temp
+mkdir -p $outdir
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)